10_Sidebar/readme_es.md

```diff
@@ -231,10 +249,11 @@ export class App extends React.Component<{}, State> {
 -        <SidebarComponent/>
 +        <SidebarComponent isVisible={this.state.isSidebarVisible}/>
         <HelloComponent userName={this.state.userName} />
-        <NameEditComponent userName={this.state.userName} onChange={this.setUsernameState.bind(this)} />
-+       <div className="float-right">
+        <NameEditComponent userName={this.state.userName} 
+            onChange={this.setUsernameState.bind(this)} 
+        />
++       <div style={{float: 'right'}}>
 +         <button
-+           className="btn btn-default"
```

Comment:
We are not using bootstrap in the samples, so we can remove this line (I think is not in the real code)

Reply:
Review this in the english version of the readme as well, thanks

Reply:
the className has been removed we have only the button tag